govcd/openapi_endpoints.go

```diff
@@ -115,7 +121,7 @@ func (client *Client) checkOpenApiEndpointCompatibility(endpoint string) (string
 // supported API versions just like client.checkOpenApiEndpointCompatibility().
 //
 // The advantage of this functions is that it provides a controlled API elevation instead of just picking the highest
-// which could be risky and untested (especially if new API version is released after release of package consuming this
+// which could be risky and untested (especially if new API version hich could be risky and untested (especially if new API version is released after release of package consuming this
```

Comment:
This remaining thing to fix.

Reply:
Rephrased that so it makes sense now. Thanks for spotting that 🚀